test/unit/BCoWPool/BCoWPool.tree

```diff
@@ -0,0 +1,12 @@
+// as we previously did in fuzzed tests, here we only test the new behaviour
+// introduced by the extension, however we don't go as far as just testing the
+// _afterFinalize hook to avoid an internal change to BPool.finalize from
+// inadvertedly breaking the derived method
```

Comment:
noted, i'd remove this comment

Reply:
hmm ok, but i don't think the comment belongs here, whatever decision we take, is taken, shouldn't be on the tree